examples/streaming-big-dataset/big_data_service.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/mysql;
+
+// Create MySQL client endpoint.
+mysql:Client testDB = new({
+        host: "localhost",
+        port: 3306,
+        name: "StreamTestDB",
+        username: "user1",
+        password: "pass1",
+        poolOptions: { maximumPoolSize: 5 }
+    });
+
+// Create a HTTP service and bind to a listener on port 9090
+service dataService on new http:Listener(9090) {
+
+    resource function getData(http:Caller caller, http:Request req) {
+        http:Response res = new;
+
+        // Invoke `select` action on the MySQL database endpoint and obtain a large amount of data.
```

Comment:
Shall we reword this to avoid action and endpoint? For ex: "Invoke `select` operation on the MySQL database client and obtain a large amount of data."